public/static/locales/en/projectDetails.json

```diff
@@ -1,3 +1,9 @@
 {
-    "satelliteAnalysis": "Satellite Analysis"
-}
\ No newline at end of file
+  "satelliteAnalysis": "Satellite Analysis",
+  "allProjects": "All Projects",
+  "searchProject": "Search Project",
+  "topProjects": "Top Projects({{noOfProjects}})",
+  "all": "All({{noOfProjects}})",
+  "list": "List",
+  "map": "Map"
+}
```

Comment:
Personally, I prefer naming the key with the purpose of the text. 

You can also group related text together so the purpose is clearer in the code.

For example,

```
"projectListTabs": {
	"all": "All({{noOfProjects}})"
	"topProjects": "Top Projects({{noOfProjects}})",
}
```

Reply:
This translation resources have been removed from this file.